hist/histv7/test/fill.cxx

```diff
@@ -5,9 +5,43 @@
 // Test Fill(), FillN(), GetEntries(), GetBinContent(), GetBinUncertainty()
 
 
-// TODO : add tests to check 0 before hand + test uncertainty after another fill
 
 
+// RHist::Fill for 1D
+
+// Test Fill() without weight and GetEntries()
+TEST(HistFillTest, FillCoordEntries) {
+  ROOT::Experimental::RH1F hist({100,0.,1});
```

Comment:
Can you use 3 spaces for indentation, please? (It might well be that some ROOT7 files still use 2 spaces... that was a failed experiment)